### ISSUE TITLE
Ability for custom functions to return structs

### ISSUE THREAD

**Issue by ralekseenkov** [Knetic/govaluate#69]
Looks like this doesn't work either after you guys implemented accessors:
"User(id).Data == '2'"

Assuming User is a custom function.

It returns an error:
Expression didn't compile correctly: User(id).Data == '2' (Unable to parse numeric value '.' to float64

**Comment by Knetic** [comment]
That's correct, this is the same problem as in #61, see [this comment](https://github.com/Knetic/govaluate/issues/61#issuecomment-321947672) for some more explanation.

This is definitely a bug, but I don't have a timeline for fixing it yet.
